src/Console/ConsoleOptionParser.php

```diff
@@ -772,65 +628,25 @@ public function parse(array $argv, ?ConsoleIo $io = null): array
     /**
      * Gets formatted help for this parser object.
      *
-     * Generates help text based on the description, options, arguments, subcommands and epilog
+     * Generates help text based on the description, options, arguments and epilog
      * in the parser.
      *
-     * @param string|null $subcommand If present and a valid subcommand that has a linked parser.
-     *    That subcommands help will be shown instead.
+     * @param string|null $subcommand unused
      * @param string $format Define the output format, can be text or XML
      * @param int $width The width to format user content to. Defaults to 72
      * @return string Generated help.
      */
     public function help(?string $subcommand = null, string $format = 'text', int $width = 72): string
     {
-        if ($subcommand === null) {
-            $formatter = new HelpFormatter($this);
-            $formatter->setAlias($this->rootName);
+        $formatter = new HelpFormatter($this);
+        $formatter->setAlias($this->rootName);
 
-            if ($format === 'text') {
-                return $formatter->text($width);
-            }
-            if ($format === 'xml') {
-                return (string)$formatter->xml();
-            }
+        if ($format === 'text') {
+            return $formatter->text($width);
+        }
+        if ($format === 'xml') {
+            return (string)$formatter->xml();
         }
-        $subcommand = (string)$subcommand;
```

Comment:
We need a return or a throw here to make respect the return types.

Reply:
Done.

Reply:
I also removed the `subcommand` param